agot-bg-game-server/src/client/game-state-panel/house-card-abilities/PatchfaceAbilityComponent.tsx

```diff
@@ -9,6 +9,8 @@ import PatchfaceAbilityGameState
 import SelectHouseCardGameState
     from "../../../common/ingame-game-state/select-house-card-game-state/SelectHouseCardGameState";
 import SelectHouseCardComponent from "../SelectHouseCardComponent";
+import SimpleChoiceGameState from "../../../common/ingame-game-state/simple-choice-game-state/SimpleChoiceGameState";
+import SimpleChoiceComponent from "../SimpleChoiceComponent";
 
 @observer
 export default class PatchfaceAbilityComponent extends Component<GameStateComponentProps<PatchfaceAbilityGameState>> {
```

Comment:
This PR makes Patchface optional but #282 also does it.

Reply:
Yea.. Sorry for confusion. I started making Patchface optional and realized that some logs are missing. So I PRed #282 and created this PR based on 282. In case you merge 282 and I rebase this this PR should only contain game logs.